subprojects/gradle/runner/client/src/test/kotlin/com/avito/runner/scheduler/runner/RunnerIntegrationTest.kt

```diff
@@ -474,6 +476,32 @@ class RunnerIntegrationTest {
             )
         }
 
+    @Test
+    fun `devices channel closed - run failed`() {
+        assertThrows<CancellationException>("Job was cancelled") {
+            runBlockingTest {
+                val devices = Channel<Device>(Channel.UNLIMITED)
+                devices.cancel()
```

Comment:
Simulate case when `Kubernetes Reservation client failed and close devices channel`